### PR TITLE
Update two_sp_queue_traffic_test.go

### PR DESCRIPTION
**Pull request by saarvind6481**

### CODE REVIEW

**Review by sezhang2 — DISMISSED** [review]
I am still working on this experimental code. Please wait.

**Comment by guoshiuan** [comment]
Internal test result: b/269796820

File: feature/experimental/qos/ate_tests/two_sp_queue_traffic_test/two_sp_queue_traffic_test.go [hunk 1]
```diff
@@ -923,7 +928,7 @@ func TestTwoSPQueueTraffic(t *testing.T) {
 		},
 		"intf1-be0": {
 			frameSize:             1000,
-			trafficRate:           60,
+			trafficRate:           50,
```
Comment (sezhang2) [inline feature/experimental/qos/ate_tests/two_sp_queue_traffic_test/two_sp_queue_traffic_test.go:931]: Why change the rate?

**Review by guoshiuan — APPROVED** [review]
We (Sean, Viktor, Cisco team and I) had a discussion (b/268403096#comment9). Viktor has filed bugs to track the explanation of Cisco implementation, and we decided to move forward with this PR.

**Review by prinikasn — COMMENTED** [review]
/gcbrun

**Review by prinikasn — APPROVED** [review]

**Pull request merged**